src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -72,7 +116,7 @@ public function loadChoiceList($value = null)
             return $this->choiceList;
         }
 
-        return $this->choiceList = new ArrayChoiceList($this->getTimezones(), $value);
+        return $this->choiceList = new ArrayChoiceList($this->getTimezones($this->regions), $value);
```

Comment:
Right :) if we can solve it simply thats nice, otherwise lets leave it out. I can validate it additionally as well, yet it's nice to have.

Btw shouldnt this be `self::getTimezones()`?